tool/jt.rb

```diff
@@ -1599,7 +1600,7 @@ def install_graal
     build
 
     dir = File.expand_path("..", TRUFFLERUBY_DIR)
-    chdir(dir) do
+    java_home = chdir(dir) do
       if LINUX
         jvmci_version = "jvmci-0.33"
```

Comment:
BTW, I think the current mac version is also `0.33` ( see below ).

Reply:
They tend to not be in sync as they come from different sources.
Probably the MAC branch should not even mention a version.

Reply:
Fixed.